.gitignore

```diff
@@ -2,3 +2,4 @@ composer.lock
 phpunit.xml
 report
 vendor
+/.idea/
```

Comment:
NOOO. Please remove this. This does NOT belong here. This MUST go in your global system ignore file.